lib/Configuration.js

```diff
@@ -102,8 +188,8 @@ export default class Configuration {
    */
   get(key, { fromFile } = {}) {
     return this.configs.find((config) => {
-      const appliesTo = config.applies_to || '**';
-      const appliesFrom = config.applies_from || '**';
+      const appliesTo = config.appliesTo || '**';
+      const appliesFrom = config.appliesFrom || '**';
```

Comment:
we could deconstruct these in the method declaration, right?

`({ appliesTo, appliesFrom }) => {...}`